pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -113,9 +116,26 @@ func TestGolden(t *testing.T) {
 			annotations: map[string]string{
 				"istio.io/insecurepath": "{\"paths\":[\"/metrics\",\"/live\"]}",
 			},
+			checkLocality: true,
+		},
+		{
+			base: "runningsds",
+			envVars: map[string]string{
+				"ISTIO_META_ISTIO_PROXY_SHA":   "istio-proxy:sha",
+				"ISTIO_META_INTERCEPTION_MODE": "REDIRECT",
+				"ISTIO_META_ISTIO_VERSION":     "release-3.1",
```

Comment:
is this version intentional?

Reply:
This is copied from the block above, I think the version is randomly picked, and this test just verifies that bootstrap config matches golden file. 

The reason I add one more test case here is because the existing one generates bootstrap config with file-mounted key/cert path, the new one generates SDS in bootstrap config.